juntagrico/templatetags/subscription.py

```diff
@@ -0,0 +1,24 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def overview(part_overview):
+    def loop(result_list, key, value, more_than_one=False):
+        if more_than_one:
+            result_list.append('<li>{}</li>'.format(key))
+            result_list.append('<ul>')
+        for new_key, new_value in value.items():
+            if isinstance(new_value, int):
+                display_name = new_key[1] if new_key[1] != '' else key
```

Comment:
This `else key` might lead to unintuitive results.
![grafik](https://user-images.githubusercontent.com/1621900/91358314-e5b27080-e7f2-11ea-8bf1-0088ff9f91b7.png)
I think the solution is to fall back to `new_key[0]` (the name) and ortoloco should set meaningful long names on the types.
Alternatively we could cut out the type name, if there is only one type in the size and show the amount a level higher.
What would work better for the other juntagrico users with their current settings?

Reply:
I strongly dissagree that cooperatives need to set meaniungfull type names if there is only one type per size
if one gets something like that they should set longnames but only then.
i do not think we will see the aove problem at the moment and if we do we know how to advise the culprits to adapt their types

Reply:
i will check if we can do this a level higher thing
because that would be the real problem this way that the size is printed twice when more than one size is present, but also this would be tolerable

Reply:
i had a look at this higher level thing, and i do not see a good solution for this since it can lead to number values on different levels which is also not a perfect solutuion
so i would suggest we leave it at that

Reply:
Ok.